<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="btnSearch">
            <property name="text">
             <string>搜索设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_user">
            <property name="text">
             <string>admin</string>
            </property>
            <property name="maxLength">
             <number>16</number>
            </property>
            <property name="placeholderText">
             <string>用户名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_passwd">
            <property name="text">
             <string>admin</string>
            </property>
            <property name="maxLength">
             <number>16</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>密码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConnect">
            <property name="text">
             <string>连接设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditUrl">
            <property name="placeholderText">
             <string>双击选择升级文件进行上传</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="btnPlayVideo">
               <property name="text">
                <string>播放视频</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="btnReboot">
               <property name="text">
                <string>重启设备</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="btnTestSpeaker">
               <property name="text">
                <string>测试喇叭</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="btnTestIperf">
               <property name="text">
                <string>测试网络性能</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="btnIrOpen">
               <property name="text">
                <string>红外灯开启</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="btnIrClose">
               <property name="text">
                <string>红外灯关闭</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="btnLedOpen">
               <property name="text">
                <string>白光灯开启</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="btnLedClose">
               <property name="text">
                <string>白光灯关闭</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="btnAlarmOpen">
               <property name="text">
                <string>指示红灯开启</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="btnAlarmClose">
               <property name="text">
                <string>指示红灯关闭</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QPushButton" name="btnAlarmBOpen">
               <property name="text">
                <string>指示蓝灯开启</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QPushButton" name="btnAlarmBClose">
               <property name="text">
                <string>指示蓝灯关闭</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QPushButton" name="btnIrcutDay">
               <property name="text">
                <string>IRCUT白天</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QPushButton" name="btnIrcutNight">
               <property name="text">
                <string>IRCUT夜晚</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QPushButton" name="btnWifiRssi">
               <property name="text">
                <string>测试WIFI信号</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QPushButton" name="btnFormatSd">
               <property name="text">
                <string>格式化SD卡</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QPushButton" name="btnPtzUp">
               <property name="text">
                <string>PTZ上移</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QPushButton" name="btnPtzDown">
               <property name="text">
                <string>PTZ下移</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QPushButton" name="btnPtzLeft">
               <property name="text">
                <string>PTZ左移</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QPushButton" name="btnPtzRight">
               <property name="text">
                <string>PTZ右移</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QPushButton" name="btnPtzReset">
               <property name="text">
                <string>PTZ复位</string>
               </property>
              </widget>
             </item>
             <item row="10" column="1">
              <widget class="QPushButton" name="btnClearConfig">
               <property name="text">
                <string>恢复出厂设置</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>640</width>
           <height>360</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <column>
       <property name="text">
        <string>MAC地址</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SN码</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP地址</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>版本号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>COMMIT</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>设备类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>UID</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
